src/rdkafka_cgrp.c

```diff
@@ -2568,6 +2568,10 @@ void rd_kafka_cgrp_handle_heartbeat_error (rd_kafka_cgrp_t *rkcg,
                 reason = "group is rebalancing";
                 break;
 
+        case RD_KAFKA_RESP_ERR_FENCED_INSTANCE_ID:
+                reason = "consumer fenced by newer instance: rebalancing";
```

Comment:
`FENCED_INSTANCE_ID` is also evidently a potential error code for JoinGroup, SyncGroup, OffsetCommit and LeaveGroup - do you want to explicitly consider it in those as well? If nothing else, for documentation purposes (I haven't checked for correctness, but assume behavior is correct without).

Reply:
is the word `: rebalancing` is redundant?